### ISSUE TITLE
Why is undetected_chromedriver automatically updated?

### ISSUE THREAD

**Issue by subingxi** [ultrafunkamsterdam/undetected-chromedriver#1996]
Why is undetected_chromedriver automatically updated?

**Comment by zero-stroke** [comment]
Are you talking about how it requires the most recently updated version of chrome to work

**Comment by subingxi** [comment]
> Are you talking about how it requires the most recently updated version of chrome to work

No, this file (/home/www/.local/share/undetected_chromedriver/undetected_chromedriver) is updated automatically. My google-chrome is not updating. As a result, it is often not used.

**Comment by crspl** [comment]
you can specify main Chrome version when creating object and then proper chromedriver will be dowloaded and used. Eg: 

`driver = uc.Chrome(headless=True, use_subprocess=False, version_main=131)`
